incubator/hnc/pkg/kubectl/tree.go

```diff
@@ -32,19 +33,35 @@ var (
 )
 
 var treeCmd = &cobra.Command{
-	Use:   "tree",
-	Short: "Displays the hierarchy tree rooted at the given namespace",
-	Args:  cobra.ExactArgs(1),
+	Use:     "tree",
+	Short:   "Displays the hierarchy tree rooted at the given namespaces separated by space and if no namespace is provided then throws an error, and in case of --all-namespace/-A flag shows tree for all root namespaces",
+	Aliases: []string{"tree"},
```

Comment:
I don't think you need this? The command is already called "tree," why did you add this line?